### ISSUE TITLE
Add embedded giveth calendar

### ISSUE THREAD

**Issue by vojtechsimetka** [Giveth/website#84]
For most people, it is difficult to find what is happening when in the giveth universe. I would like to see the Giveth calendar to be added on the website somewhere and to update the calendar with all the events and meetings we are going to/attending.

Here is the iframe:
```
<iframe src="https://calendar.google.com/calendar/embed?src=givethdotio%40gmail.com style="border: 0" width="800" height="600" frameborder="0" scrolling="no"></iframe>
```
and direct link: 
https://calendar.google.com/calendar/embed?src=givethdotio%40gmail.com&ctz=Europe%2FPrague

**Comment by geleeroyale** [comment]
I like the idea - not sure if it fits in the current website concept (presentation), so I suggest to add this to the wiki (ressources and information) - We can have a dedicated events/calendar page there or just put it in on the welcome site.

**Comment by vojtechsimetka** [comment]
I agree, would be even cooler if we could, on top of having whole giveth calendar, filter per person/group of people. I'll keep dreaming.
